admin_pages/general_settings/templates/state_details_settings.template.php

```diff
@@ -73,15 +73,15 @@
             <input class="STA_abbrev small-text "
                    id='STA_abbrev-XXX'
                    name="STA_abbrev_XXX"
-                   title=""
+                   aria-label=""
                    type="text"
                    value=""
             />
         </td>
         <td class="general-settings-country-state-input-td">
             <label for="STA_name_XXX"><?php esc_html_e('Name', 'event_espresso'); ?></label>
             <br/>
-            <input id="STA_name-XXX" class="STA_name regular-text " type="text" title="" value="" name="STA_name_XXX">
+            <input id="STA_name-XXX" class="STA_name regular-text " type="text" aria-label="" value="" name="STA_name_XXX">
```

Comment:
I did some research and it is not recommended to have empty aria-label as its main use case is for screen readers.

At least, we can put the label title there.

Reply:
fixed